sktools/sk

```diff
@@ -0,0 +1,214 @@
+command="$1"
+
+if ! [ $command == "start" -o $command == "stop" -o $command == "status" \
```

Comment:
This sort of command selection is usually easier to manage with a switch statement in bash.  Calls to per command functions go in the switch statement, and the default case is the "error" case.